### PR TITLE
Fixes gun funnies

### PR DESCRIPTION
**Pull request by Bod9001**
turns out gun code is messy, tested with editor and client

### Changelog:

CL: [Fix] Guns now play audio when firing
CL: [Fix] guns now spawn bookcases if they can
CL: [Fix] guns now spawn Bullet casings if they can
CL: [Fix] guns now have cool down before firing again

### CODE REVIEW

**Comment by corp-0** [comment]
bookcases?

**Comment by Bod9001** [comment]
bookcases xD haha ok I am adding that

**Comment by corp-0** [comment]
No but for real, what did you mean?

**Comment by Bod9001** [comment]
bullet casing

**Comment by Bod9001** [comment]
image
think it would be okay to add this? xD

File: UnityProject/Assets/Scripts/Items/Weapons/Gun.cs [hunk 1]
```diff
@@ -607,8 +532,10 @@ void ProgressFinishAction()
 					$"You begin removing the {FiringPin.gameObject.ExpensiveName()} from {gameObject.ExpensiveName()}",
 					$"{interaction.Performer.ExpensiveName()} begins removing the {FiringPin.gameObject.ExpensiveName()} from {gameObject.ExpensiveName()}.");
 
-				AudioSourceParameters audioSourceParameters = new AudioSourceParameters(UnityEngine.Random.Range(0.8f, 1.2f));
-				SoundManager.PlayNetworkedAtPos(CommonSounds.Instance.WireCutter, interaction.Performer.AssumedWorldPosServer(), audioSourceParameters, sourceObj: serverHolder);
+				AudioSourceParameters audioSourceParameters =
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Weapons/Gun.cs:535]: these are the cases for what ``var`` is for.

**Review by corp-0 — APPROVED** [review]

**Pull request merged**